snapshots/demography/2023-10-10/riley_2005.csv

```diff
@@ -0,0 +1,60 @@
+year,entity,life_expectancy

```

Comment:
You can delete the file

Reply:
I actually need it to run the snapshot step, see `riley_2005.py`

Reply:
I know, but I thought that in general we don't want to keep local CSV files after uploading them. As long as we keep the instructions to get it in the code